sources/packages/backend/apps/api/src/app.exception.filter.ts

```diff
@@ -1,24 +1,24 @@
-import { Catch, ArgumentsHost } from "@nestjs/common";
+import { Catch, ArgumentsHost, Injectable } from "@nestjs/common";
 import { BaseExceptionFilter } from "@nestjs/core";
 import { Request } from "express";
-import { LoggerService, InjectLogger } from "@sims/utilities/logger";
+import { LoggerService } from "@sims/utilities/logger";
 
 @Catch()
+@Injectable()
```

Comment:
Asking for clarification, is the injectable decorator required here? (I may have been misled seeing the nest example that `@Catch` alone must be sufficient.

Reply:
It is redundant; just tested and removed it.